src/components/UI/Display/MAPI/AccessControl/AccessControlRoleListItem.tsx

```diff
@@ -49,53 +48,117 @@ interface IAccessControlRoleListItemProps
+      active,
+      permissions,
+      groups,
+      users,
+      serviceAccounts,
+      ...props
+    },
+    ref
+  ) => {
+    const groupCount = Object.values(groups).length;
+    const groupCountFormatted = formatCounter(groupCount);
+    const userCount = Object.values(users).length;
+    const userCountFormatted = formatCounter(userCount);
+    const serviceAccountCount = Object.values(serviceAccounts).length;
+    const serviceAccountCountFormatted = formatCounter(serviceAccountCount);
 
-  return (
-    <StyledCard
-      pad={{ vertical: 'small', horizontal: 'medium' }}
-      round='xsmall'
-      elevation='none'
-      overflow='visible'
-      background='background-front'
-      aria-selected={active}
-      role='button'
-      tabIndex={active ? -1 : 0}
-      aria-label={name}
-      {...props}
-      ref={ref}
-    >
-      <CardHeader>
-        <StyledHeading level={5} margin='none'>
-          <Box direction='row' align='center'>
-            <Text truncate={true}>{name}</Text>
+    const hasBoundSubjects =
+      groupCount > 0 || userCount > 0 || serviceAccountCount > 0;
 
-            {namespace.length < 1 && (
-              <Text margin={{ left: 'xxsmall' }}>
-                <i
-                  className='fa fa-globe'
-                  role='presentation'
-                  aria-label='Cluster role'
-                />
-              </Text>
-            )}
-          </Box>
-        </StyledHeading>
-      </CardHeader>
-      <CardBody margin={{ top: 'xsmall' }}>
-        <Box justify='between' direction='row'>
-          <Box width='100px'>
-            <Text color='text-weak'>Groups: {groupCount}</Text>
-          </Box>
-          <Box width='100px'>
-            <Text color='text-weak'>Users: {userCount}</Text>
-          </Box>
-        </Box>
-      </CardBody>
-    </StyledCard>
-  );
-});
+    return (
+      <StyledCard
+        pad={{ vertical: 'small', horizontal: 'medium' }}
+        round='xsmall'
+        elevation='none'
+        overflow='visible'
+        background='background-front'
+        aria-selected={active}
+        role='button'
+        tabIndex={active ? -1 : 0}
+        aria-label={name}
+        {...props}
+        ref={ref}
+      >
+        <CardHeader>
+          <StyledHeading level={5} margin='none'>
+            <Box direction='row' align='center'>
+              <Text truncate={true}>{name}</Text>
+
+              {namespace.length < 1 && (
+                <Text margin={{ left: 'xxsmall' }}>
+                  <i
+                    className='fa fa-globe'
+                    role='presentation'
+                    aria-label='Cluster role'
+                  />
+                </Text>
+              )}
+            </Box>
+          </StyledHeading>
+        </CardHeader>
+        {hasBoundSubjects && (
+          <CardBody margin={{ top: 'xsmall' }}>
+            <Box direction='row'>
+              {groupCount > 0 && (
+                <Box width='30%'>
```

Comment:
What do you think about getting rid of these wrapper elements `<Box width='30%'>`? We can separate children with some fixed gap, for example. I think it could make the layout more independent of the childs count.